apps/frontend/src/components/Auth/apiClient.tsx

```diff
@@ -86,17 +93,55 @@ export class ApiClient {
     return this.get("/auth/timesheet") as Promise<string>;
   }
 
+
+  // a function to create user object from a passed in userId
+  public async createUser(userId: String): Promise<UserSchema> {
+    var userConverted = {}
+
+    await this.get(`/user/usersById?userIds[]=${userId}`).then((userList) => {
```

Comment:
similar to my previous comment, this might need to be wrapped in a try catch

Reply:
I agree with Ethan - we don't have error handling in frontend or backend right now, but best practice (until we have specific error codes being returned from the backend) is just to wrap in a try catch for now.